deployer/deployer.py

```diff
@@ -473,7 +484,7 @@ def deploy(self, environment='pre', eureka=True, health_check=None, health_check
         self.elbs = []
         if self.elb:
             if not self.loadbalancer_exist():
-                if not self.create_loadbalancer():
+                if not self.create_loadbalancer(health_check, health_check_port):
```

Comment:
@padilo  Since now the health_check and health_check_port parameters are only for deploys without eureka although this parameters can used without deploys without eureka I think this parameters must have a default values (not here, in the create_loadbalancer function)